cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -2,74 +2,109 @@
+        {%- endset %}
+    {% endif %}
 
-        {% if latest_version_url %}
-            <div class="ons-container">
-                {% call onsPanel({"variant": 'warn', "classes": "ons-u-mt-s"}) %}
-                    <p>{{ _("This has been superseded.") }} <a href="{{ latest_version_url }}">{{ _("View corrected version") }}</a></p>
-                {% endcall %}
-            </div>
-        {% endif %}
+    {% set next_release_date_text = page.next_release_date|date("DATE_FORMAT") or _("To be announced") %}
 
-        {% block release_meta %}
-            {% if page.is_census %}
-                <div class="ons-container">
-                    {% include "templates/components/census/census-logo.html" %}
-                </div>
-            {% endif %}
-            <div class="ons-container common-header__releases">
-                <div>
-                    <span class="ons-u-fs-r--b common-header__releases-label">{{ _("Release date") }}:</span>
-                    <span class="ons-u-nowrap">{{ page.release_date|date("DATE_FORMAT") }}</span>
-                </div>
-                <div>
-                    <span class="ons-u-fs-r--b common-header__releases-label">{{ _("Next release") }}:</span>
-                    <span class="ons-u-nowrap">{{ page.next_release_date|date("DATE_FORMAT") or _("To be announced") }}</span>
-                </div>
-                <div>
-                    <span class="ons-u-fs-r--b common-header__releases-label">{{ _("Edition") }}:</span>
-                    <span class="ons-u-nowrap">{{ _("Latest") }}</span>
-                </div>
-                <div>
-                    <span class="ons-u-fs-r--b common-header__releases-label">{{ _("Releases") }}:</span>
-                    {% if page.is_latest %}
-                        <a href="{{ routablepageurl(page.get_parent().specific, "previous_releases") }}">{{ _("View previous releases") }}</a>
-                    {% else %}
-                        <a href="{{ routablepageurl(page.get_parent().specific, "latest_release") }}">{{ _("View latest release") }}</a>
-                    {% endif %}
-                </div>
-                {% if page.contact_details %}
-                    <div>
-                        <span class="ons-u-fs-r--b common-header__releases-label">{{ _("Contact") }}:</span>
-                        <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.name }}</a>
-                    </div>
-                {% endif %}
-            </div>
-        {% endblock %}
-    </div>
+    {# fmt:off #}
+    {{
+        onsHero({
+            "topic": _("Statistical article"),
+            "title": page.display_title,
+            "variants": 'grey',
+            "text": page.summary,
+            "officialStatisticsBadge": page.is_accredited,
+            "wide": true,
+            "detailsColumns": 12,
+            "censusLogo": page.is_census,
```

Comment:
This doesn't seem to be working as there are no changes to the hero, when is census is ticked. Does this fall under the follow up action point of `Further design updates for the hero are in progress with DS team so this may need updating again later on`, or a bug in DS?

Reply:
It also worked fine for me when I tested.

Reply:
Had marked it as resolved after I posted. User error, I didn't load the latest DS templates.